packages/example-foundry/foundry.toml

```diff
@@ -0,0 +1,27 @@
+[profile.default]
+src = "src"
+out = "artifacts"
+libs = ["lib"]
+remappings = [
+    "@openzeppelin/contracts-upgradeable/=node_modules/@openzeppelin/contracts-upgradeable/",
+    "@openzeppelin/contracts/=node_modules/@openzeppelin/contracts/",
+    "ds-test/=lib/forge-std/lib/ds-test/src/",
+    "forge-std/=lib/forge-std/src/",
+]
+
+# OPTIMIZER SETTINGS
+optimizer = true
+optimizer_runs = 200
+
+# RPC SETTINGS
+
+[rpc_endpoints]
+default = "http://localhost:8545"
+mumbai = "https://rpc-mumbai.maticvigil.com/"
+
+# ETHERSCAN SETTINGS
+
+[etherscan]
+mumbai = { key = "YOUR_API_KEY_HERE" }
+
+# See more config options https://github.com/foundry-rs/foundry/blob/master/crates/config/README.md#all-options
```

Comment:
If this file contains secrets maybe is better to rename it to `foundry.toml.example` and add  `foundry.toml` to git ignore file

Reply:
We can import env secrets into `foundry.toml` there is no need to create `.example`.
I will update it slightly so its shown how to do it.